frontend/src/routes.js

```diff
@@ -137,6 +140,27 @@ const routes = [
         requiresAuth: true,
         title: 'Statping - Service Failures',
       }
+    },{
+      path: 'downtimes',
+      component: DashboardDowntimes,
+      meta: {
+          requiresAuth: true,
+          title: 'Statping - Downtimes',
+      }
+    },{
+      path: 'create_downtime',
+      component: EditDowntime,
```

Comment:
intentional to render `EditDowntime` component here? because same is rendered on `/edit_downtime`

Reply:
ya create and edit downtime is having same component to render